tools/schematics/dt-component/files/libs/barista-components/__name@dasherize__/jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
please create a `jest.config.json` like the button.

Reply:
Added it, does that mean I should remove the `jest.config.js`?

Reply:
yep